JSDemos/Demos/Scheduler/Timelines/description.md

```diff
@@ -1 +1,10 @@
-This demo illustrates the collection of&nbsp;timeline views available in&nbsp;the Scheduler component. Appointments are associated with several resources (an&nbsp;appointment color depends on&nbsp;the associated resource) and grouped by&nbsp;one of&nbsp;them.
\ No newline at end of file
+Timeline views display appointments as a sequence of events on a horizontal timeline, as opposed to basic views that display appointments in a calendar-like fashion. You can use the View Switcher above the timetable to switch between the views.
+
+To add timeline views to your Scheduler, list them in the [views](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array. To activate a specific view at application startup, use the [currentView](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#currentView) property.
```

Comment:
I'd like to switch the order of the first sentence, since it is the first sentence in the paragraph (and the second sentence  also starts with (To...):

```suggestion
List timeline views in the [views](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/) array to add them to your Scheduler. To activate a specific view at application startup, use the [currentView](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#currentView) property.
```